Web Quiz Engine (Kotlin)/task/src/engine/WebQuizRepository.kt

```diff
@@ -8,10 +10,7 @@ import java.util.concurrent.ConcurrentMap
 typealias QuizIdentifier = Int
 
 @Component
```

Comment:
Should use the `@Repository` annotation instead.
It's functionally the same but it's a better way to organize your Beans in the project.

Reply:
True that. I did it before I learned there were such a specialized annotation hehe.
Done

Reply:
Quick question, any idea why there are no `CrudRepository` annotation like there is both a `Controller` and `RestController` one?

Reply:
Not sure about that really unfortunately. I'll ask CoPilot real quick.